test/integration/noderesourcetopology_test.go

```diff
@@ -553,6 +553,59 @@ func TestTopologyMatchPlugin(t *testing.T) {
 			},
 			expectedNodes: []string{"fake-node-1"},
 		},
+		{
+			name: "Scheduling Best-Effort pod with most-allocatable strategy scheduler",
```

Comment:
I think we should run a test like this also for `leastAllocatable` and `balancedAllocation`.